<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_question</class>
 <widget class="QDialog" name="dialog_question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#dialog_question{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox#groupBox {
	border: 2px solid #000000;
	
	background-color: rgb(250, 250, 210);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,10,70,10">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="25,75">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
border: 1px solid #000000;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="25,75">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Вариант:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
border: 1px solid #000000;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="25,75">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Текст задания:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="styleSheet">
           <string notr="true">QTextEdit {
border: 1px solid #000000;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,40,20,20">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(249, 250, 150);
border: 1px solid #000000;
border-radius: 20px;
} 
QPushButton::hover { 
	
	background-color: rgb(227, 227, 136);
} 
QPushButton::pressed {
 border: 3px solid #000000;
}</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(249, 250, 150);
border: 1px solid #000000;
border-radius: 20px;
} 
QPushButton::hover { 
	
	background-color: rgb(227, 227, 136);
} 
QPushButton::pressed {
 border: 3px solid #000000;
}</string>
          </property>
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(249, 250, 150);
border: 1px solid #000000;
border-radius: 20px;
} 
QPushButton::hover { 
	
	background-color: rgb(227, 227, 136);
} 
QPushButton::pressed {
 border: 3px solid #000000;
}</string>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
